src/mame/drivers/generalplus_gpl32612.cpp

```diff
@@ -157,6 +185,54 @@ void generalplus_gpl32612_game_state::gpl32612(machine_config &config)
 	SPEAKER(config, "rspeaker").front_right();
 }
 
+void generalplus_zippity_game_state::machine_start()
+{
+	generalplus_gpl32612_game_state::machine_start();
+
+	// if there's a cart, override the standard mapping
+	if (m_cart && m_cart->exists())
+	{
+		m_cart_region = memregion(std::string(m_cart->tag()) + GENERIC_ROM_REGION_TAG);
+	}
+}
+
+DEVICE_IMAGE_LOAD_MEMBER(generalplus_zippity_game_state::cart_load)
+{
+	uint32_t size = m_cart->common_get_size("rom");
+
+	m_cart->rom_alloc(size, GENERIC_ROM16_WIDTH, ENDIANNESS_LITTLE);
+	m_cart->common_load_rom(m_cart->get_rom_base(), size, "rom");
+
+	return image_init_result::PASS;
+}
+
+
+
+void generalplus_zippity_game_state::zippity(machine_config &config)
+{
+	// don't inherit from gpl32612 as in reality this is GPL32300A and could differ
+
+	ARM9(config, m_maincpu, 240000000); // unknown core / frequency, but ARM based
```

Comment:
Please use digit grouping with big numbers so the order of magnitude can be seen at a glance.  Counting the zeroes to see if this is 2.4GHz, 240MHz or 24MHz is slow.